benefits/in_person/forms.py

```diff
@@ -26,7 +26,7 @@ def __init__(self, agency: models.TransitAgency, *args, **kwargs):
         super().__init__(*args, **kwargs)
         flows = agency.enrollment_flows.filter(supported_enrollment_methods__contains=models.EnrollmentMethods.IN_PERSON)
 
-        self.classes = "checkbox-parent"
+        self.classes = "in-person-eligibility-form"
```

Comment:
I decided to add the `in-person` here because I want to be able to know for sure I'm working in Admin. It can get confusing when both files are named `style.css` and have a classes that have similar names.